test/quantization/test_quantized_op.py

```diff
@@ -1565,6 +1571,92 @@ def test_batch_norm2d(self, X, Y_scale, Y_zero_point):
             quantize_ref = torch.quantize_per_tensor(float_ref, Y_scale, Y_zero_point, dtype_x)
             self.assertEqual(qy.int_repr().numpy(), quantize_ref.int_repr().numpy())
 
+    @given(batches=st.integers(1, 16),
+           num_groups=st.sampled_from((1, 2, 3, 4, 6, 8, 12)),
+           channels_per_group=st.sampled_from((1, 2, 3, 4, 7, 9, 16, 32)),
+           elements_per_channel=st.integers(2, 1024),
+           torch_type=st.sampled_from((torch.qint8, torch.quint8, torch.qint32)),
+           X_rand_scale=st.floats(0.01, 1e3),
+           Y_scale=st.floats(0.2, 2.6),
+           Y_zero_point=st.integers(0, 5))
+    def test_group_norm(self, batches, num_groups, channels_per_group,
+                        elements_per_channel, torch_type, X_rand_scale,
+                        Y_scale, Y_zero_point):
+        if "fbgemm" not in torch.backends.quantized.supported_engines:
+            return
+
+        with override_quantized_engine("fbgemm"):
+            num_channels = num_groups * channels_per_group
+            shapes = (batches, num_channels, elements_per_channel)
+
+            # In the FP kernel, mean and variance are calculated in floating point.
+            # In the quantized kernel, they are calculated in integer arithmetic.
+            # Because of this, the numerics do not always match exactly which is
```

Comment:
Can we also point out that the quantized computation is exact, as we do not incur any precision loss.